src/Symfony/Component/HttpClient/NativeHttpClient.php

```diff
@@ -133,14 +133,14 @@ public function request(string $method, string $url, array $options = []): Respo
 
         // Always register a notification callback to compute live stats about the response
         $notification = static function (int $code, int $severity, ?string $msg, int $msgCode, int $dlNow, int $dlSize) use ($onProgress, &$info) {
-            $now = microtime(true);
-            $info['total_time'] = $now - $info['start_time'];
+            $info['total_time'] = microtime(true) - $info['start_time'];
```

Comment:
Just for information, why not using the stopwatch here?

Reply:
"Why use it?" would my answer :)
More seriously, this backports something that is provided by curl natively. We don't want nor need to provide Stopwatch to curl when it already does everything without us doing anything. This provides "same behavior" to NativeHttpClient, to make them as alike as possible.

Reply:
Thanks